index.html

```diff
@@ -319,13 +319,7 @@ <h2 class="title-bullet-points">Soft Skills</h2>
           </div>
         </section>
 
-        <section class="member-imgs" id="members">
-          <img class="member-img member_animation" alt="member-img" />
-          <img class="member-img member_animation" alt="member-img" />
-          <img class="member-img member_animation" alt="member-img" />
-          <img class="member-img member_animation" alt="member-img" />
-          <img class="member-img member_animation" alt="member-img" />
-        </section>
+        <section class="member-imgs" id="members"></section>
```

Comment:
Empty tags can be self closing
`<section class=.......... />`

Reply:
We are dynamically adding tags later...So we need the closing tag here!